src/components/ComparisonDetailMap/ComparisonDetailMap.vue

```diff
@@ -41,7 +45,15 @@
           customAttribution: MAP_CUSTOM_ATTRIBUTIONS,
         },
         map: undefined,
-        isLoadingSatelliteImage: false,
+        generatingSatelliteImageUrl: {
+          loading: {
+            state: false,
+          },
+          error: {
+            state: false,
+            message: '',
+          },
+        },
```

Comment:
Similar to what we use here https://github.com/global-water-watch/global-water-watch-website/blob/main/src/pages/reservoir/_slug.vue#L72